TraceScreen.c

```
@@ -34,6 +35,8 @@ static const char* const TraceScreenKeys[] = {"F3", "F4", "F8", "F9", "Esc"};
 
 static const int TraceScreenEvents[] = {KEY_F(3), KEY_F(4), KEY_F(8), KEY_F(9), 27};
 
+static ProgramLauncher TraceScreen_programLauncher;
+
```

Cppcheck (2.20.0)

[style] The function 'Vector_quickSort' is never used.

(unusedFunction)